### DISCUSSION TITLE
Since Swiper v10 `swiper/bundle` import fails (with Parcel)

### DISCUSSION THREAD

**Discussion by onetrev** [nolimits4web/swiper#6860] [Q&A]
Ever since the _package_ code was refactored with v10, I'm finding that I can no longer use the **bundle** import if using Parcel as your bundler. With Swiper v9 I'm able to do: `import Swiper from 'swiper/bundle';` as per the documentation. But if that with Swiper v10, I get this error:

```
@parcel/core: Failed to resolve 'swiper/bundle' from './dev/js/scripts.js'

  /wp-content/themes/custom-theme/dev/js/scripts.js:10:20
     9 | // import Swiper bundle with all modules installed
  > 10 | import Swiper from 'swiper/bundle';
  >    |                    ^^^^^^^^^^^^^^^
    11 | 
    12 | // init Swiper:

@parcel/resolver-default: Cannot load file './bundle' from module 'swiper'
```

For now, it's not a big deal, but you have to add each module individually as described in the docs:
```
// core version + navigation, pagination modules:
import Swiper from 'swiper';
import { Navigation, Pagination } from 'swiper/modules'
```
And then when you init your Swiper instance you have to add: ` modules: [Navigation, Pagination],`

Does anyone know a workaround for this though? Parcel is a pretty major bundling tool, so it feels like this should work, especially when it used to previously. However, this doesn't feel like this should be a bug / issue so I'm positing it here -- as much as I'd like to see it fixed. :)

**Comment by dadoadk** [comment]
I'm trying the same with Angular Ionic, there's no way to include pagination. When I try importing as you described above I get a compilation error. Does somebody use the newest Swipper 10 in Angular? Let me know how you got the pagination working.

> ng] × Failed to compile.
> [ng] Error: node_modules/swiper/types/modules/index.d.ts:1:35 - error TS2691: An import path cannot end with a '.d.ts' extension. Consider importing '../shared.js' instead.
> [ng] 1 import type { SwiperModule } from '../shared.d.ts';

**Reply by dadoadk** [reply]
I got the fix by using Elements. 

// import function to register Swiper custom elements
import { register } from 'swiper/element/bundle';
// register Swiper custom elements
register();

**Reply by onetrev** [reply]
Cool, glad you got it fixed. I think your issue is a bit different from mine, maybe, but your fix seemed similar to my workaround as well. 

My issue was not being able to include the **entire** bundle. I was like you able to include modules as need be.
